libsplinter/src/node_id/store/error.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::error::Error;
+use std::fmt::Display;
+
+#[derive(Debug)]
+/// Error for NodeIdStores
+pub enum NodeIdStoreError {}
```

Comment:
This error enum should include at least one variant; if its not obvious what the variants will be, a safe bet is that InternalError(InternalError) is one of them.

Reply:
Updated/fixed